src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -87,12 +92,18 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
    * Currently selected phase split
    */
   selectedPhaseSplit: any = null;
+  showLeaderboardByLatest = false;
 
   /**
    * Sort leaderboard based on this column
    */
   sortColumn = 'rank';
 
+  /**
+   * Text option for leadeboard sort
+   */
+  sortLeaderboardTextOption = 'Sort by latest';
```

Comment:
Can we type cast here just `sortLeaderboardTextOption: string;`? As we gonna initliaze when we select phase.

Reply:
Changed!